### PR TITLE
sys/net: add VAINA configuration interface

### PR DESCRIPTION
**Pull request by jeandudey**
This adds a configuration interface to add clients to the Router Client Set, in the future the VAINA (**vaina** as of now) will be extended to support other features, it can only be used over SLIP, messages should be sent to if local multicast address `ff01::1`

VAINA stands for **VAINA - Versatile Address Interface | Network Administration**.

### CODE REVIEW

File: sys/net/vaina/vaina.c [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+                return -ENOSPC;
+            }
+            break;
+
+        case VAINA_MSG_RCS_DEL:
+            aodvv2_client_delete(&msg->payload.del.ip);
+            break;
+#endif
+
+        default:
```
Comment (luisan00) [inline sys/net/vaina/vaina.c:96]: default case should raise an event error or something to return

Comment (jeandudey) [reply sys/net/vaina/vaina.c:96]: Done.

File: main.c [hunk 2]
```diff
@@ -57,6 +63,27 @@ int main(void)
...
+        if (ipv6_addr_from_str(&addr, SLIP_LOCAL_ADDR) == NULL) {
+            printf("Malformed SLIP local address, please verify it!\n");
+        }
+
+        if (gnrc_netif_ipv6_addr_add(slipdev_netif, &addr, 128, 0) != sizeof(ipv6_addr_t)) {
+            printf("Couldn't setup SLIP local address\n");
+        }
+
+        if (vaina_init(slipdev_netif) < 0) {
+            printf("Coudln't initialize VAINA");
```
Comment (luisan00) [inline main.c:82]:
```suggestion
            printf("Couldn't initialize VAINA\n");
```

**Comment by luisan00** [comment]
configuring a custom `SLIP LOCAL_ADDR` should be an option in the` menuconfig`

**Comment by jeandudey** [comment]
>configuring a custom `SLIP_LOCAL_ADDR` should be an option in the menuconfig

I'll try to do it, there are some limitations by having it in menuconfig

**Comment by jeandudey** [comment]
Comments addressed, please review.

**Comment by luisan00** [comment]
Travis looks unhappy =(

**Comment by jeandudey** [comment]
@luisan00 looks like the connection died when downloading GCC, I restarted the build and I hope [It will survive](https://www.youtube.com/watch?v=ARt9HV9T0w8) next time.

**Comment by luisan00** [comment]
I will pray for him

**Comment by jeandudey** [comment]
Commits squashed for easy review of each one.

**Pull request merged**